apps/backend/src/modules/users/user.entity.ts

```diff
@@ -81,47 +97,57 @@ class UserEntity implements Entity {
 		passwordSalt: string;
 	}): UserEntity {
 		return new UserEntity({
+			allowNotifications: "false",
```

Comment:
Let's move "false", "true" values to a separate enum. It'll help us when we add other values

Reply:
Should it be shared or backend only?

Reply:
I think you used those values in Frontend as well, so in shared